templates/islandora-solr-metadata-display.tpl.php

```diff
@@ -15,7 +15,7 @@
  */
 ?>
 
-<?php if (count($solr_fields) > 1): ?>
+<?php if (count($solr_fields) > 0): ?>
```

Comment:
Actually, looking at the history of this file, [it was started](https://github.com/discoverygarden/uws-theme/commits/master/templates/islandora-solr-metadata-display.tpl.php) before the [present `!empty()` check was added](https://github.com/Islandora/islandora_solr_metadata/commit/27d7a9964d8880b366ce563786f8ca7983edc9e6#diff-1db44dfa40f6183fdca4208d583f968fR17)... So `> 1` might just be an off-by-one... Possibly supposed to be `>=`? Anyway...